anclafx/provide.go

```diff
@@ -13,14 +14,13 @@ const Module = "ancla"
 func Provide() fx.Option {
 	return fx.Module(
 		Module,
-		fx.Invoke(chrysom.ProvideStartListenerClient),
+		fx.Invoke(listener.ProvideStartListenerClient),
 		fx.Provide(
 			ancla.ProvideService,
 			ancla.ProvideListener,
 			ancla.ProvideDefaultListenerWatchers,
-			chrysom.ProvideBasicClient,
-			chrysom.ProvideDefaultListenerReader,
-			chrysom.ProvideListenerClient,
+			listener.ProvideReaderOption,
+			listener.ProvideListenerClient,
```

Comment:
moved listener related code to its own package because we introduced options (listener and client have their own sets of options)